editoast/migrations/2023-10-11-222214_delete_track_links/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Your SQL goes here
+
+DROP TABLE infra_layer_track_section_link;
+DROP TABLE infra_object_track_section_link;
```

Comment:
Since we are removing `InfraErrorType::OverlappingTrackLinks` we should delete errors of this type.

```sql
DELETE FROM infra_layer_error WHERE information->>'obj_type' = 'TrackSectionLink';
```